avocado/utils/cpu.py

```diff
@@ -213,14 +213,17 @@ def get_cpuidle_state():
     :rtype: Dict of dicts
     """
     cpus_list = cpu_online_list()
-    states = range(len(glob.glob("/sys/devices/system/cpu/cpu0/cpuidle/state*")))
+    states = range(
```

Comment:
What is the pep8 issue you're seeing?  When I, (and Travis CI) runs `./selftests/inspekt-style.sh` I don't see any problems.  Also by running manyally:

```
$ inspekt style --exclude=.git --disable E501,E265,W601,E402,E722 avocado/utils/cpu.py
PEP8 disabled: E501,E265,W601,E402,E722
PEP8 compliance check PASS
```

Reply:
i ran pep8 so it showed  me pep8 issues , than i ran `autopep8 -i` which made these changed , if you say i will squash   this commit  please suggest